Sources/Bluetooth/L2CAPSocket.swift

```diff
@@ -9,7 +9,7 @@
 import Foundation
 
 /// L2CAP Socket protocol.
-public protocol L2CAPSocketProtocol: class {
+public protocol L2CAPSocketProtocol: AnyObject {
```

Comment:
This is a separate issue from the `allCases`. I find it hard to believe simply declaring a `class` protocol is broken or has been changed in Swift 5.4. If there has been a breaking language change please link to the Swift Evolution proposal or discussion in Forum.